internal/convert/deck.go

```diff
@@ -78,7 +78,6 @@ func getDeckChangelogConfig(_ *config.Hal) *config.Deck_Changelog {
 func getDeckFeaturesConfig(h *config.Hal) *config.Deck_Features {
 	return &config.Deck_Features{
 		PipelineTemplates:            h.GetFeatures().GetPipelineTemplates(),
-		Canary:                       h.GetFeatures().GetMineCanary(),
```

Comment:
Does this mean we can also remove the `canary` field from the `Deck_Features` proto?
https://github.com/spinnaker/kleat/blob/1d42fc37545346fb0f26b63c532c203540531e49/api/proto/config/deck.proto#L57

Reply:
Great catch, thank you! 🤗